static/app/components/profiling/flamegraph/flamegraph.tsx

```diff
@@ -355,8 +355,8 @@ function Flamegraph(props: FlamegraphProps): ReactElement {
             />
           </ProfileDragDropImport>
         }
-        frameStack={
-          <FrameStack
+        flamechartDrawer={
```

Comment:
This is still calling it flamechart and the props above. Don't have a strong opinion in either direction, but if we can stick with 1 that'd be great.

Reply:
Yup, let me rename that too